storage/scratchpad/benches/sparse_merkle.rs

```diff
@@ -62,10 +63,11 @@ impl Group {
                 b.iter_batched(
                     || small_batches.clone(),
                     |small_batches| {
-                        block
+                        let ret = block
                             .smt
                             .batches_update(small_batches, &block.proof_reader)
                             .unwrap();
+                        ret // return the result smt so that the Drop time is not counted
```

Comment:
just remove the `;`?

Reply:
I feel this is more explicit (easy mistake to make to add the semicolon)

Reply:
this?
|small_batches| -> SparseMerkleTree {
}

Reply:
as you wish (see the fixup commit)